example/ck_tile/37_elementwise/reference_add.hpp

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2023, Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
Please update copyright headers in all files.
```suggestion
// Copyright (c) 2025, Advanced Micro Devices, Inc. All rights reserved.
```

Reply:
Fixed in commit bcccb723.